models/comments.js

```diff
@@ -0,0 +1,38 @@
+const { Model } = require('sequelize')
+
+module.exports = (sequelize, DataTypes) => {
+    class Comments extends Model {
+        /**
+         * Helper method for defining associations.
+         * This method is not a part of Sequelize lifecycle.
+         * The `models/index` file will call this method automatically.
+         */
+        static associate(models) {
+            Comments.belongsTo(models.User, {
+                as: 'users',
+            })
+            Comments.belongsTo(models.News, { as: 'news', })
+        }
+    }
+    Comments.init({
+        userId: {
+            type: DataTypes.INTEGER,
+            allowNull: false
+        },
+        newsId: {
+            type: DataTypes.INTEGER,
+            allowNull: false
```

Comment:
In the migration, you set the newId to null when the related new is deleted. But in the model you put that it does not accept nulls. Coordinate that.

Reply:
fixed